upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -2387,10 +2387,10 @@ func describeInstanceType(c AWSCloud, instanceType string) (*ec2.InstanceTypeInf
 	}
 	resp, err := c.EC2().DescribeInstanceTypes(req)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error describing instance type %q in region %q: %w", instanceType, c.Region(), err)
```

Comment:
error strings should not start with the word "error" as they are printed with "Error: " prepended.

```suggestion
		return nil, fmt.Errorf("describing instance type %q in region %q: %w", instanceType, c.Region(), err)
```